rfc/rfc012-nuts-company-credential.md

```diff
@@ -0,0 +1,119 @@
+# RFC012 Nuts Company Credential
+
+|  |  |
+| :--- | :--- |
+| Nuts foundation | W.M. Slakhorst |
+| Request for Comments: 012 | Nedap |
+|  | February 2021 |
+
+## Nuts Company Credential
+### Abstract
+
+Creating a network with trusted Verified Credentials is victim of the chicken-and-egg-problem. 
+Issuers are not yet convinced they should support VCs until the network is mature enough and nobody is willing to use the network without official credentials/identities.
+The **Nuts company credential** offers a temporary solution. It allows for any DID subject to issue a VC. 
```

Comment:
As in: it allows for any DID subject to make claims about its own identity?

Reply:
or others, yes